src/lib/openpath/openpath.js

```diff
@@ -75,7 +75,11 @@ export const fetchTrips = async (client, account, startDate) => {
   const tripsMetadata = await fetchTripsMetadata(token, startDate, {
     excludeFirst: !firstRun
   })
-  logService('info', `Trips metadata ${JSON.stringify(tripsMetadata)}`)
+  if (!tripsMetadata || tripsMetadata.length < 1) {
```

Comment:
question nit: Why `< 1` and not `=== 0` ? .length is an unsigned integer so can't be < 0.

Reply:
it's an old habit learnt in my first programming lessons: prefer range comparisons in case of float traps. So this is not useful here, but I always do like this, as a no brainer